tests/e2e/default/analytics/config/targets-error-config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = [
+  {
+    id: 'muted-this-month',
+    context: 'user.parent.type === "district_hospital"',
+    type: 'count',
+    goal: 0,
+    appliesTo: 'contacts',
+    appliesIf: (c) => !!c.foo.muted
+  },
+];
```

Comment:
What do you think about creating a `const` with the name `muted` for example, and then exporting that `const`? in that way will be easier for other people to add new config values and to understand the meaning of that specific config.

I know there are a lot of config files where the code is exactly as you did it, and it is not wrong, is just that we are trying to organize a little bit more the tests code in general 🙂 

You can see this file as an example: [tests/factories/cht/reports/generic-report.js](https://github.com/jonathanbataire/cht-core/blob/8039-error-log/tests/factories/cht/reports/generic-report.js)